tests/zfs-tests/tests/functional/cli_root/zpool_scrub/zpool_scrub_offline_device.ksh

```diff
@@ -49,7 +49,7 @@ verify_runnable "global"
 
 function cleanup
 {
-	poolexists $TESTPOOL && destroy_pool $TESTPOOL
+	poolexists $TESTPOOL2 && destroy_pool $TESTPOOL2
```

Comment:
Why was the testpool name changed for this test?

Reply:
Otherwise, this test destroys the pool created by `setup.sh`, which breaks any tests that come after it (like `zpool_scrub_multiple_copies.ksh`).

Reply:
Got it